auth/auth-router.js

```diff
@@ -1,11 +1,55 @@
-const router = require('express').Router();
+const router = require("express").Router();
+const Users = require("./authHelper");
+const secrets = require("../config/secrets");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
 
-router.post('/register', (req, res) => {
-  // implement registration
+router.post("/register", async (req, res) => {
+  let user = req.body;
+  console.log(user);
+  const hash = bcrypt.hashSync(user.password, 10);
+  user.password = hash;
+  const addedUser = await Users.add(user);
+
+  try {
+    if (addedUser) {
+      if (!addedUser.username) {
+        res.status(401).json({ message: "please provide username" });
+      } else {
+        res.status(201).json(addedUser);
+      }
+    }
+  } catch (error) {
+    res.status(500).json(error);
+  }
 });
 
-router.post('/login', (req, res) => {
-  // implement login
+router.post("/login", async (req, res) => {
+  let { username, password } = req.body;
+  const user = await Users.findBy({ username }).first();
```

Comment:
Again, you want a check to make sure the user sent all of the required information. And the await function should go inside of the try.